src/legacy/Routes.tsx

```diff
@@ -33,13 +33,64 @@ import Transactions from './components/Transactions'
 import CompanionAppListener from './companion'
 import { PastelID } from '../features/pastelID'
 import WormholeConnection from './components/WormholeConnection'
-import { connect } from 'react-redux'
+import { connect, useSelector } from 'react-redux'
 import { setPastelConf } from '../features/pastelConf'
+import { pastelDBSelector } from '../redux/modules/pastelDB'
+import { getLastIdFromDB, getStatisticDatasFromDB, insertStatisticDataToDB } from '../legacy/utils/PastelDB'
+
+interface PastelDBThreadProps {
+  rpc: any;
+  rpcConfig: any;
+}
+
+const PastelDBThread = (props: PastelDBThreadProps) => {
```

Comment:
According to our guidelines, this should live in the `src/features/<your feature>` directory